src/views/blocks/tickets.php

```diff
@@ -133,7 +133,19 @@
 		$this->template( 'blocks/tickets/item-inactive', [ 'is_sale_past' => $is_sale_past ] );
 	endif;
 
-	$this->template( 'v2/components/loader/loader' );
+	ob_start();
+	/**
+	 * Allows filtering of extra classes used on the tickets-block loader
+	 *
+	 * @since  4.11.0
+	 *
+	 * @param  array $classes The array of classes that will be filtered.
+	 */
+	$loader_classes = apply_filters( 'tribe_tickets_block_loader_classes', [ 'tribe-tickets-loader__tickets-block' ] );
+	include Tribe__Tickets__Templates::get_template_hierarchy( 'components/loader.php' );
```

Comment:
Why this change?

Reply:
I'm just rolling back to what we had on tickets v1. This is the template of tickets v1.

The inclusion of the loader component v2 here was merged when we didn't decide to split v1/v2 yet. So, at some point we changed this on the bucket branch, but to me it doesn't make sense to move into the loader v2 in the tickets view v1. That's why I'm reverting this. And also because it affects the snapshots (the loader in v1 includes an extra class, coming from the `tribe_tickets_block_loader_classes ` filter).

Reply:
👍